config/samples/udg2/udg_external_graphql_internal_rest.yaml

```diff
@@ -0,0 +1,105 @@
+apiVersion: tyk.tyk.io/v1alpha1
+kind: ApiDefinition
+metadata:
+  name: rest-countries
+spec:
+  name: rest-countries
+  use_keyless: true
+  protocol: http
+  active: true
+  proxy:
+    target_url: https://restcountries.com/v2/alpha/
+    listen_path: /restCountries
+    strip_listen_path: true
+  version_data:
+    default_version: Default
+    not_versioned: true
+    versions:
+      Default:
+        name: Default
+
+
+
+---
+
+
+apiVersion: tyk.tyk.io/v1alpha1
+kind: ApiDefinition
+metadata:
+  name: udg
+spec:
+  name: Universal Data Graph v2b
+  use_keyless: true
+  protocol: http
+  active: true
+  proxy:
+    target_url: ""
+    listen_path: /udg
+    strip_listen_path: true
+  version_data:
+    default_version: Default
+    not_versioned: true
+    versions:
+      Default:
+        name: Default
+  graphql:
+    enabled: true
+    execution_mode: executionEngine
+    schema: |
+      type Country {
+        name: String
+        code: String
+        restCountry: RestCountry
+      }
+
+      type Query {
+        countries: [Country]
+      }
+
+      type RestCountry {
+        altSpellings: [String]
+        subregion: String
+        population: Int
+      }
+    version: "2"
+    last_schema_update: "2022-10-12T14:27:55.511+03:00"
+    type_field_configurations: []
+    playground:
+      enabled: true
+      path: /playground
+    engine:
+      field_configs:
+        - disable_default_mapping: false
+          field_name: countries
+          path:
+            - "countries"
+          type_name: Query
+        - disable_default_mapping: true #very important for rest APIs
```

Comment:
It might be better to explain it's importance for REST APIs. Is there any external documentation about it? If not, it might be better to replace
```yaml
#very important for rest APIs
```
with a small explanation about importance of `disable_default_mapping` field

Reply:
There is external documentation and I feel it's in the court of the graphQL time to provide explanations to this given that it does not affect to operator logic in any way.